plugins/optimization-detective/tests/test-optimization.php

```diff
@@ -368,44 +368,6 @@ function ( OD_HTML_Tag_Walker $walker, OD_URL_Metrics_Group_Collection $url_metr
 		$this->assertEquals( $expected, $buffer );
 	}
 
-	/**
-	 * Test get_response_header().
-	 *
-	 * @covers OD_Preload_Link_Collection::get_response_header
-	 */
-	public function test_get_response_header(): void {
-		$collection = new OD_Preload_Link_Collection();
-
-		$collection->add_link(
-			array(
-				'href'          => 'https://example.com/foo.jpg',
-				'as'            => 'image',
-				'fetchpriority' => 'high',
-				'imagesrcset'   => 'https://example.com/foo-480w.jpg 480w, https://example.com/foo-800w.jpg 800w',
-				'imagesizes'    => '(max-width: 600px) 480px, 800px',
-				'crossorigin'   => 'anonymous',
-			),
-			null,
-			null
-		);
-
-		$collection->add_link(
-			array(
-				'href'          => 'https://example.com/bar.jpg',
-				'as'            => 'image',
-				'fetchpriority' => 'high',
-				'imagesrcset'   => 'https://example.com/"bar"-480w.jpg 480w, https://example.com/"bar"-800w.jpg 800w',
-				'imagesizes'    => '(max-width: 600px) 480px, 800px',
-				'crossorigin'   => 'anonymous',
-			),
-			600,
-			1200
-		);
-
-		$expected_header = 'Link: <https://example.com/foo.jpg>; rel="preload"; as="image"; fetchpriority="high"; imagesrcset="https://example.com/foo-480w.jpg 480w, https://example.com/foo-800w.jpg 800w"; imagesizes="(max-width: 600px) 480px, 800px"; crossorigin="anonymous"; media="screen", <https://example.com/bar.jpg>; rel="preload"; as="image"; fetchpriority="high"; imagesrcset="https://example.com/\"bar\"-480w.jpg 480w, https://example.com/\"bar\"-800w.jpg 800w"; imagesizes="(max-width: 600px) 480px, 800px"; crossorigin="anonymous"; media="screen and (min-width: 600px) and (max-width: 1200px)"';
-		$this->assertSame( $expected_header, $collection->get_response_header() );
-	}
-
```

Comment:
Assertions were incorporated into an existing test in `test-class-od-html-tag-processor.php`